frb_codegen/src/generator/dart/mod.rs

```diff
@@ -467,53 +620,106 @@ fn generate_api_func(func: &IrFunc, ir_file: &IrFile) -> GeneratedApiFunc {
     }
 }
 
-fn generate_api2wire_func(ty: &IrType, ir_file: &IrFile, block_index: BlockIndex) -> String {
-    if let Some(body) = TypeDartGenerator::new(ty.clone(), ir_file, None).api2wire_body(block_index)
-    {
+fn rust_wire_to_dart_wire(ty: &str) -> Cow<str> {
+    let ty = ty.trim();
+    if matches!(ty, "void" | "WireSyncReturnStruct") {
+        return ty.into();
+    }
+    if ty.starts_with("*mut") {
+        return format!("int /* {} */", ty).into();
+    }
+    return format!("dynamic /* {} */", ty).into();
```

Comment:
Curious: what was the type before this function is introduced in the PR? All dynamic, or all typed?

If all dynamic - this PR is great, improving static type checks :)